BaseCollector.py

```diff
@@ -86,6 +87,11 @@ def get_collection_times(self):
         self.collection_times = request.json()
         return self.collection_times
 
+    def get_inventory_api_response(self):
+        request = requests.get(url="http://" + os.environ['INVENTORY'] + "/api_response_code")
```

Comment:
This will get the response code of the last thread who just wrote something to `self.api_response`.
It maybe would make sense by target, but even then it is just the last request we are seeing here. Not sure if that makes sense.

Reply:
You are right. Changed it to a dict similar to collection times.